packages/client/src/formValidation.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  emailValidation: function (email) {
+    if (
+      /^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/.test(
+        email
+      )
+    ) {
+      return null;
```

Comment:
We probably want to  keep it consistent here, either return `null` in all of our validators or `undefined` in all of our validators.

Reply:
Noticed the different returns during testing and wondered, but totally missed why that was the case!  That's what you get for c/p code lol.  Now it makes sense.  TY!